examples/2-query/bsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "2-query",
   "reason": {
-    "react-jsx": 2
+    "react-jsx": 3
   },
+  "bsc-flags": ["-bs-super-errors"],
```

Comment:
Awesome, thanks for adding this! I forgot to track back after getting the initial `bsconfig`s setup, but this is wonderful to have 🚀